packages/react-components/src/Autocomplete/autocomplete.tsx

```diff
@@ -0,0 +1,686 @@
+  padding: '10px 0',
+});
+
+const AutocompleteDropdownGroupName = styled(Typography)(
+  (props) => ({
+    padding: 'var(--pv-size-base-2)',
+    color: props.theme.mode === 'dark'
+      ? 'var(--pv-color-gray-6)'
+      : 'var(--pv-color-gray-9)',
+  }),
+);
+
+const AutocompleteDropdownGroupList = styled('ul')({
+  padding: 0,
+  listStyleType: 'none',
+});
+
+const AutocompleteDropdownGroupListItem = styled(MenuItem)<Required<{ inGroup: boolean }>>(
+  (props) => ({
+    ...(props.inGroup && {
+      padding: '0px var(--pv-size-base-2) 0 var(--pv-size-base-3)',
+    }),
+  }),
+);
+
+const AutocompletePopover = styled(Popper)(
+  {
+    minWidth: 240,
+    outline: 0,
+    marginTop: '1px',
+    borderRadius: '4px',
+    minHeight: '16px',
+    maxHeight: 'calc(100% - 32px)',
+    zIndex: 1300,
+  },
+  (props) => {
+    const isDark = props.theme.mode === 'dark';
+
+    let backgroundColor = 'var(--pv-color-white)';
+    let boxShadow = 'var(--pv-shadow-light-low)';
+
+    if (isDark) {
+      backgroundColor = 'var(--pv-color-gray-3)';
+      boxShadow = 'var(--pv-shadow-dark-medium)';
+    }
+
+    return ({
+      backgroundColor,
+      boxShadow,
+    });
+  },
+);
+
+const AutocompleteTag = styled(Chip)<{
+  tagsLength: number,
+  limitTags: number,
+  size: AutocompleteOwnProps<any>['size'],
+}>((props) => ({
+  label: 'Autocomplete-tag',
+  borderRadius: '3px',
+  margin: 0,
+  ...(props.tagsLength === 1 && {
+    maxWidth: 'calc(100% - var(--pv-size-base))',
+  }),
+  ...(props.tagsLength > 1 && props.limitTags > 0 && {
+    maxWidth: `calc(${100 / props.limitTags}% - var(--pv-size-base))`,
+  }),
+  ...(props.size === 'small' && {
+    height: 'var(--pv-size-base-5)',
+  }),
+}));
+
+const AutocompleteTagSize = styled(Typography)({
+  margin: '0 var(--pv-size-base-2)',
+});
+
+const AutocompleteInputField = styled(Typography)(
+  () => ({
+    fontFamily: 'inherit',
+    outline: 'none',
+    boxSizing: 'border-box',
+    minWidth: '30px',
+    width: 0,
+    flexGrow: 1,
+    backgroundColor: 'transparent',
+    borderStyle: 'none',
+    appearance: 'none',
+  }),
+  (props) => {
+    const isDark = props.theme.mode === 'dark';
+    const color = isDark
+      ? 'var(--pv-color-white)'
+      : 'var(--pv-color-black)';
+
+    let colorPlaceholder = 'var(--pv-color-gray-9)';
+    let colorDisabled = 'var(--pv-color-gray-7)';
+
+    if (isDark) {
+      colorPlaceholder = 'var(--pv-color-gray-6)';
+      colorDisabled = 'var(--pv-color-gray-4)';
+    }
+
+    return ({
+      color,
+      '&::placeholder': {
+        color: colorPlaceholder,
+      },
+      '&:disabled': {
+        cursor: 'not-allowed',
+        color: colorDisabled,
+      },
+    });
+  },
+);
+
+const AutocompleteError = styled(Typography)({
+  marginTop: '2px',
+});
+
+const AutocompleteLabel = styled('label')({
+  label: 'TextField-label',
+  marginBottom: '2px',
+  display: 'inline-block',
+});
+/**
+ *
+ */
+
+export const Autocomplete = <
```

Comment:
Check the `clear` button. There is in the mockups, but there is no implementation.

<img width="568" alt="Screenshot 2024-09-30 at 12 01 42" src="https://github.com/user-attachments/assets/f4cdc572-6718-4b6a-b62b-35ee9acc6cec">